code/frameworks/angular/package.json

```diff
@@ -91,17 +91,17 @@
     "zone.js": "^0.11.0"
   },
   "peerDependencies": {
-    "@angular-devkit/architect": ">=0.1300.0",
-    "@angular-devkit/build-angular": ">=13.0.0",
-    "@angular-devkit/core": ">=13.0.0",
-    "@angular/cli": ">=13.0.0",
-    "@angular/common": ">=13.0.0",
-    "@angular/compiler": ">=13.0.0",
-    "@angular/compiler-cli": ">=13.0.0",
-    "@angular/core": ">=13.0.0",
-    "@angular/forms": ">=13.0.0",
-    "@angular/platform-browser": ">=13.0.0",
-    "@angular/platform-browser-dynamic": ">=13.0.0",
+    "@angular-devkit/architect": ">=0.1500.0",
+    "@angular-devkit/build-angular": ">=15.0.0",
+    "@angular-devkit/core": ">=15.0.0",
+    "@angular/cli": ">=15.0.0",
+    "@angular/common": ">=15.0.0",
+    "@angular/compiler": ">=15.0.0",
+    "@angular/compiler-cli": ">=15.0.0",
+    "@angular/core": ">=15.0.0",
+    "@angular/forms": ">=15.0.0",
+    "@angular/platform-browser": ">=15.0.0",
+    "@angular/platform-browser-dynamic": ">=15.0.0",
```

Comment:
If this works...
I assume we're going to have to think about how to make this backwards compatible?